tests/unit/core/test_searchUtils.js

```diff
@@ -1,100 +1,64 @@
-import createStore from 'amo/store';
-import * as searchActions from 'core/actions/search';
-import * as api from 'core/api';
-import { initialApiState } from 'core/reducers/api';
-import { ADDON_TYPE_THEME } from 'core/constants';
-import { loadByCategoryIfNeeded, mapStateToProps } from 'core/searchUtils';
+import {
+  isLoaded,
+  parsePage,
+} from 'core/searchUtils';
 
 
-describe('searchUtils mapStateToProps()', () => {
+describe('Search.isLoaded()', () => {
```

Comment:
I know we're lax about module level descriptions but I'm not sure what `Search` is in this context. Can it be something that will be more helpful when viewed in the console? I think `core/searchUtils/isLoaded()` would be helpful but `isLoaded` would be fine too. The problem is that jest doesn't always provide useful stack traces (e.g. wonky event loop stuff).